wordclock/settings.h

```diff
@@ -66,7 +65,18 @@ typedef struct {
     uint32_t magic_end;
 } TSettings;
 
+// The HW settings are static for a certain hardware build.
+// This prevents unreadable clocks when TSettings is updated (13x13 clock is unreadale when set to 11x11)
+// By splitting this settings, TSettings can freely be changed and assigned new defaults,
+typedef struct {
+    uint32_t magic;
+    EHardwareType hardwareType;
+    EPixelType pixelType;
+    uint8_t reserved[256];
```

Comment:
Waarvoor is deze reserved?

Reply:
Zodat je velden kan toevoegen zonder dat de magic aangepast moet worden